iml-wire-types/src/lib.rs

```diff
@@ -1523,6 +1523,71 @@ pub struct ResourceAgentInfo {
     pub agent: ResourceAgentType,
     pub id: String,
     pub args: HashMap<String, String>,
+    pub ops: PacemakerOperations,
+}
+
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum OrderingKind {
+    Mandatory,
+    Optional,
+    Serialize,
+}
+
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum PacemakerScore {
+    Infinity,
+    Value(i32),
+    NegInfinity,
+}
+
+#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct PacemakerOperations {
+    // Time to wait for Resource to start
+    pub start: Option<String>,
+    // Time of monitor interval
+    pub monitor: Option<String>,
+    // Time to wait for Resource to stop
+    pub stop: Option<String>,
+}
+
+impl PacemakerOperations {
+    pub fn new<'a>(
```

Comment:
It doesn't look like any borrows are being used. You should be able to remove the lifetime parameter, right?

Reply:
that's a copy/paste error, yes.